HISTORY.md

```diff
@@ -1,4 +1,4 @@
-unreleased
+1.7.5 / 2024-10-21
```

Comment:
Remember to update this when we do publish.

Reply:
Oh yeah!

Reply:
Updated!